isoCluster.py

```diff
@@ -0,0 +1,279 @@
+import sys
+import os
+from pathlib import Path
+import shutil
+import ipaddress
+from dataclasses import dataclass
+import re
+import time
+import urllib.parse
+from concurrent.futures import Future
+from concurrent.futures import ThreadPoolExecutor
+from typing import Dict
+from typing import List
+from typing import Tuple
+from logger import logger
+from clustersConfig import ClustersConfig
+from clustersConfig import NodeConfig
+from clustersConfig import ExtraConfigArgs
+import host
+import common
+
+
+"""
+ExtraConfigIPU is used to provision and IPUs specified via Redfish through the IMC.
+This works by making some assumptions about the current state of the IPU:
+- The IMC is on MeV 1.2 / Mev 1.3
+- BMD_CONF has been set to allow for iso Boot
+- ISCSI attempt has been added to allow for booting into the installed media
+- The specified ISO contains full installation kickstart / kargs required for automated boot
```

Comment:
I wonder if it would be helpful to verify the ISO in the script. I.e. mounting and checking contents. Sanity check before spending the time to debug.

Reply:
Yeah probably a good idea, I can add this in a follow up PR